docs/deployment/README.md

```diff
@@ -54,9 +54,9 @@ The process involves:
 
 ## Installation
 
-#### Prepare DAAC deployment repository {#prepare-deployment}
+#### Prepare DAAC deployment repository
 
-_If you already are working with an existing `<daac>-deploy` repository that is configured appropriately for the version of Cumulus you intend to deploy or update, skip to [Prepare AWS configuration. ](#prepare-config)_
+_If you already are working with an existing `<daac>-deploy` repository that is configured appropriately for the version of Cumulus you intend to deploy or update, skip to [Prepare AWS configuration. ](#prepare-aws-configuration) _
```

Comment:
I'm currently not able to run the documentation website locally, but in the GitHub preview, the `_` characters aren't formatting the text... I think this might be due to a space between the `)` and last `_`